qa/build.gradle

```diff
@@ -45,7 +45,9 @@ dependencies {
     testRuntimeOnly('com.jayway.jsonpath:json-path:2.9.0') {
         // OpenSearch core is using slf4j 1.7.36. Therefore, we cannot change the version here.
         exclude group: 'org.slf4j', module: 'slf4j-api'
+        exclude group: 'net.minidev', module: 'json-smart'
     }
+    testRuntimeOnly group: 'net.minidev', name:'json-smart', version: "${versions.json_smart}"
```

Comment:
you need to rebase on latest main and remove this

Reply:
My [fork branch](https://github.com/q-andy/neural-search-os/tree/neural-stats) is up to date with main with latest commits, I think this diff is from the [feature branch](https://github.com/opensearch-project/neural-search/tree/feature/neural-stats-api.) on this repo being out of date. I don't have write permissions to sync it, could you try syncing it and this should disappear?